.github/workflows/gradle-build-development.yml

```diff
@@ -20,12 +20,14 @@ jobs:
         run: chmod +x gradlew
       - name: Build with Gradle
         run: ./gradlew assemble
+        env:
+          - REACT_APP_API_URL: https://checkins-develop-tuvcfzotpq-uc.a.run.app
```

Comment:
@jahanner The dash on the front of this failed the build: https://github.com/oci-labs/check-ins/actions/runs/258147232

See: https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables

Can you get a fix in for this please?